actions-cleanup-openstack-vms/readme.md

```diff
@@ -0,0 +1,28 @@
+# Overview
+
+This action is clean up VMs that are older than a certain period of time in openstack
+
+# Example
+
+```yaml
+name: "clean up idle VMs in OVH openstack"
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  cleanup:
+    runs-on: [self-hosted, centos7, large]
+    steps:
+      - name: Clean up orphan VMs in ${{ env.CLOUD }}
```

Comment:
And now we talk about "orphans". Let's align everything.

Reply:
ok. I would try to be consistent.